pkg/server/errors_test.go

```diff
@@ -23,20 +23,20 @@ import (
 
 	"github.com/codenotary/immudb/embedded/store"
 	immuerrors "github.com/codenotary/immudb/pkg/errors"
-	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 )
 
 func TestMapServerError(t *testing.T) {
 	err := mapServerError(store.ErrIllegalState)
-	assert.Equal(t, ErrIllegalState, err)
+	require.Equal(t, ErrIllegalState, err)
```

Comment:
it may be better to improve error comparison with `require.ErrorIs(t, err, ErrIllegalState)`

Reply:
I see that there's [this issue](https://github.com/codenotary/immudb/issues/1435) that relates to your remarks. I can take on this issue too, but should I make such changes on this branch?

Reply:
Sure, in another PR would be fine. Thanks a lot

Reply:
And is this PR considered ok, or should I just wait for review? I don't really know what should happen next, so sorry if that's a dumb question.